pkgs/by-name/fa/fahrplan/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3, fetchFromGitHub }:
+let
+  pname = "fahrplan";
+  owner = "dbrgn";
+  version = "1.1.2";
+in python3.pkgs.buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-2QVaA2zqqFuxjkKbDXo+SNzkrDCxXiSYMUOftQ++wO4=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [ dateutil requests texttable ];
```

Comment:
```suggestion
  dependencies = with python3.pkgs; [ dateutil requests texttable ];
```

Reply:
Out of curiosity, why is it done this way? The docs say to use propagatedBuildInputs

Reply:
No they don't: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters